### ISSUE TITLE
[BLOCKED] [PACT Act Wizard] PACT Act launch plan

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17083]
## Status
**[2024-09-18] [Fran]** Stakeholders have mothballed PAW, however we will keep the code just in case it's resurrected. Closing this ticket as a no op. It can be reopened in future if need be.
**[2024-12-03] [Fran]** Blocked until stakeholders determine path forward for PAW
**[2024-08-20] [Jill]** Final VA reviewers have asked for more time to review. Latest request will mean final content should be available for dev between 9.2 and 9.6. Once ready - will need to be parked in staging for review by VA Secretary and White House (no content change opportunity). This can be done in parallel with final QA/etc. [Slack thread](https://dsva.slack.com/archives/C079GFKN50D/p1724168710585209?thread_ts=1724082034.558829&cid=C079GFKN50D)
**[2024-08-08] [Fran]** Contacted Kristin and Danielle regarding the entrypoints for PAW (ticket [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65776#issuecomment-2276310474)).
**[2024-02-08 Fran]** Per Dave Conlon on 2/7, the stakeholders wish to make content changes, so the launch is delayed (more than likely 2nd Q 2024)

## Project Tracking
-  [Link](https://agilesix.slack.com/lists/T46Q2J7QV/F07QZLX6R3L?view_id=View07RNEU0WN4)

## User Story

**AS A** Veteran
**I WANT** you to launch the PACT Act Wizard
**SO THAT** I can use it to determine if I am eligible for VA benefits and/or compensation

## Pre-launch
- [ ] Full QA has been conducted by team members assigned (TBD), and passed
- [x] CAIA Staging Review has occurred and feedback presented
  - [ ] CAIA Staging review [launch blocking feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73575) is implemented
- [x] QA Standards have been met according to this [ticket](https://app.zenhub.com/workspaces/vagov-sitewide-product-team-6645154188a81a1f7a0a9464/issues/gh/department-of-veterans-affairs/va.gov-team/73518)
  - [x] Note that the lack of a Traceability Matrix is not launch blocking; Fran has one started, but it's very convoluted and will be a work in progress.
- [x] Privacy, Security, and Readiness Review [ticket](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/issues/1831) submitted
  - [x] Privacy, Security, and Readiness Review passed
- [ ] Collab cycle ticket (s) have been addressed/closed
- [ ] Analytics ticket has been created 
- [ ] Content and flow are verified
  - [ ] Greenlight from Danielle that all updated text has been provided and implemented
  - [ ] CAIA has successfully QA'd  and Passed all content on Staging 
  - [ ] PW team members tagged for QA have successfully tested and Passed all logic flows according to [Mural logic flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692989444688/0044b9825c82d8d23920601f68c41a61d047d681?sender=u0b235d03cbd64f7f93673243)
- [ ] Share targeted launch day with Danielle T so she can coordinate implementing entry point (ticket [#65776](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65776#issuecomment-2277978127)

## Go / No Go 
- [ ] Greenlight from Amanda  Klausmeier for launch
- [ ] Greenlight from Michelle Middaugh for launch
- [ ] Greenlight from Danielle Thierry for launch

## Launch
- [ ] Content-build PR 2236 must be merged **after 10am ET but before COB the day before launch** in order to actually deploy the PACT Act app to prod: https://github.com/department-of-veterans-affairs/content-build/pull/2236. 
- [ ] Content-build deploys successfully, including our commit ([Deploy job](https://github.com/department-of-veterans-affairs/content-build/actions/workflows/daily-production-release.yml); (Deployment [schedules ](https://depo-platform-documentation.scrollhelp.site/developer-docs/deployments)on VA Platform).
- [ ] Contact Danielle T so the entry point can be implemented. Ticket [here](PACT Act IA
[#65776](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65776).
- [ ] App is live at https://www.va.gov/pact-act-eligibility 
  - [ ] Let Cindy know!
- [ ] Any socialization/announcements needed have been sent
- [ ] Ensure all associated collab cycle and epic tickets are closed

## Day After Launch Day
- [ ] Entry points have been implemented / released to PROD via ticket [#65776](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65776#issuecomment-2277978127)
- [ ] 508 Audit Request has been filed #18896 
- [ ] Add info to ToT PW slide for socialization - remember Party Cone!)

**Comment by randimays** [comment]
This content-build PR must be merged in order to actually deploy the PACT Act app to prod: https://github.com/department-of-veterans-affairs/content-build/pull/2236

Once it is merged, that code will follow the normal deploy cadence for content-build into production. The app will be live at `https://www.va.gov/pact-act-eligibility` once the changes make it to prod.

**Comment by jilladams** [comment]
@FranECross I looked over launch plan - great work here! I made a couple edits (Feel free to revise if they don't make sense for you): 
1. Nested the content / flow verification steps under a parent. (Nitpick, just made it easier to scan for me.)
2. Moved the greenlights to a "Go / No go" section that should happen before we merge the PR to expose it in prod
3. Changed the timeframe for merging the PR to "after 10am ET but before COB the day before launch". Technically, we _can_ merge anytime up til 10a ET on launch day, BUT: in order to deploy, after we merge, our commit has to pass CI on main (I'm pretty sure that's true in content-build like it is in vets-website). If CI has trouble (which it often does), merging morning-of could be unnecessarily risky. If we can merge by COB the night before, it'll give us some breathing room, just in case.

Again: feel free to edit / change any of that if you have strong feelings.

**Comment by FranECross** [comment]
@jilladams Love the changes! I'll mirror on the DUW launch plan, and will also create a launch plan template, using generic terms where we mentioned specific products.

**Comment by jilladams** [comment]
FYI that Cindy pinged me today to see if we had news about when this may launch. I told her we'd let her know when it does, and to ensure that I remember to do so, I've added it to the launch plan.

**Comment by jilladams** [comment]
Final VA reviewers have asked for more time to review. Latest request will mean final content should be available for dev between 9.2 and 9.6.
Once ready - will need to be parked in staging for
review by VA Secretary and White House (no content change opportunity). This can be done in parallel with final QA/etc.
https://dsva.slack.com/archives/C079GFKN50D/p1724168710585209?thread_ts=1724082034.558829&cid=C079GFKN50D
